freud/order.pyx

```diff
@@ -740,6 +740,31 @@ cdef class LocalQl(Compute):
     def __str__(self):
         return repr(self)
 
+    @Compute._computed_method()
+    def plot(self, ax=None):
+        """Plot Ql.
+
+        Args:
+            ax (:class:`matplotlib.axes`): Axis to plot on. If :code:`None`,
+                make a new figure and axis. (Default value = :code:`None`)
+
+        Returns:
+            (:class:`matplotlib.axes`): Axis with the plot.
+        """
+        import plot
+        return plot.histogram_plot(self.Ql,
+                                   title="Local Ql",
+                                   xlabel=r"$Ql$",
```

Comment:
Make the _x_ label like `"$Q_4$"` - insert the actual value of `l`. Same for the title, and for other methods.